src/QMCHamiltonians/tests/test_QMCHamiltonian.cpp

```diff
@@ -42,6 +53,224 @@ TEST_CASE("QMCHamiltonian::flex_evaluate", "[hamiltonian]")
+  if constexpr (generate_test_data)
+  {
+    std::vector<Real> vector_kinetic;
+    std::copy(kinetic.begin(), kinetic.end(), std::back_inserter(vector_kinetic));
+    std::cout << " size kinetic: " << vector_kinetic.size() << '\n';
+    std::cout << " std::vector<Real> kinetic_ref_vector = " << NativePrint(vector_kinetic) << ";\n";
+
+    std::vector<Real> vector_pots;
+    std::copy(local_pots.begin(), local_pots.end(), std::back_inserter(vector_pots));
+    std::cout << " size potentials: " << vector_pots.size() << '\n';
+    std::cout << " std::vector<Real> potential_ref_vector = " << NativePrint(vector_pots) << ";\n";
+
+    std::vector<Real> vector_ions;
+    std::copy(ion_pots.begin(), ion_pots.end(), std::back_inserter(vector_ions));
+    std::cout << " size ion potentials: " << vector_ions.size() << '\n';
+    std::cout << " std::vector<Real> ion_potential_ref_vector = " << NativePrint(vector_ions) << ";\n";
+  }
+  else
+  {
+    std::vector<Real> kinetic_ref_vector = {
+        -0.5, -0, -0,   -0, -0, -0, -0, -0, -0, -0.5, -0, -0,   -0, -0, -0, -0,
+        -0,   -0, -0.5, -0, -0, -0, -0, -0, -0, -0,   -0, -0.5, -0, -0, -0, -0,
+    };
+    std::vector<Real> potential_ref_vector = {
+        -0.4304472804, -0.2378402054, -0.9860844016, -0.08201235533, -0.8099648952, -0.9536881447, -0.4498806596,
+        -0.4415832162, -0.9407452345, -0.6449454427, -2.240605593,   -1.194514155,  0.04097786546, -0.1860728562,
+        -0.1374063194, -0.8808210492, 0.3020428121,  0.3772183955,   -0.112801373,  -0.5531326532, 0.4633262753,
+        0.3185032904,  0.3001851439,  -0.4555109739, -0.2190704495,  -0.7140043378, -1.641614318,  0.1718038917,
+        -0.7621642947, 0.2606962323,  -0.1486036181, -1.001747012,
+    };
+    std::vector<Real> ion_potential_ref_vector = {
+        0.04332125187, 0.173753351, -0.9332901239, -1.323815107, 1.695662975, 0.5990064144, 0.1634206772, -1.207304001,
+    };
+
+    std::size_t num_data = num_electrons * num_walkers;
+    for (std::size_t id = 0; id < num_data; ++id)
+    {
+      INFO("id : " << id);
+      CHECK(kinetic_ref_vector[id] == Approx(kinetic(id)));
+    }
+
+    for (std::size_t id = 0; id < num_data; ++id)
+    {
+      INFO("id : " << id);
+      CHECK(potential_ref_vector[id] == Approx(local_pots(id)));
+    }
+
+    std::size_t num_data_ions = num_ions * num_walkers;
+    for (std::size_t id = 0; id < num_data_ions; ++id)
+    {
+      INFO("id : " << id);
+      CHECK(ion_potential_ref_vector[id] == Approx(ion_pots(id)));
+    }
+
+    // When only EE and EI coulomb potentials the local energy is just the sum of
+    // the local_pots and kinetic
+    auto sum_local_pots = std::accumulate(local_pots.begin(), local_pots.end(), 0.0);
+    auto sum_kinetic    = std::accumulate(kinetic.begin(), kinetic.end(), 0.0);
+    auto sum_local_nrg  = std::accumulate(local_nrg.begin(), local_nrg.end(), 0.0);
+    CHECK(sum_local_nrg == Approx(sum_local_pots + sum_kinetic));
+  }
+  outputManager.resume();
```

Comment:
Remove this resume and the pause above. We should have suppressed the output by default in unit tests.